ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -236,6 +239,9 @@ const CreateRunPage = () => {
       },
       evaluator: {
         enabled: true,
+        copyrightGarbageFile: '',
+        licenseClassificationsFile: '',
+        ruleSet: '',
```

Comment:
Let's align with the other names and call this `rulesFile`. Also because the term "rule*Set*" reminds me too much of `evaluator/src/main/kotlin/RuleSet.kt` which is a class that *can* be used as part of rules, but does not *have* to be.

In general, I think it's a good idea to look at the respective CLI tool's options and their help when choosing names and descriptions. So in this case, I'd look at [these](https://github.com/oss-review-toolkit/ort/blob/146ab3bc6e5f790d304e1d6f4187f072961d8541/plugins/commands/evaluator/src/main/kotlin/EvaluatorCommand.kt#L110-L139).

Reply:
`ruleSet` comes from back-end API and cannot be changed in front-end.

Reply:
@mnonnenmacher do you think it makes sense to rename that backend property?

Reply:
We had this discussion internally before but could not agree on another name. `ruleSetFile` and `rulesFile` would both be fine with me, but mainly I would like if the suffix was aligned with the other properties.

Reply:
> I would like if the suffix was aligned with the other properties.

Exactly my thought as well.